http/console/controller_make_command.go

```diff
@@ -61,5 +67,26 @@ func (receiver *ControllerMakeCommand) populateStub(stub string, name string) st
 func (receiver *ControllerMakeCommand) getPath(name string) string {
 	pwd, _ := os.Getwd()
 
-	return pwd + "/app/http/controllers/" + str.Camel2Case(name) + ".go"
+	controllerName, _, folderPath := parseName(name)
+
+	if folderPath != "" {
+		folderPath = folderPath + "/"
+	}
+
+	return pwd + "/app/http/controllers/" + folderPath + str.Camel2Case(controllerName) + ".go"
+}
+
+func parseName(name string) (string, string, string) {
+
+	parts := strings.Split(name, "/")
+
+	controllerName := parts[len(parts)-1]
+	packageName := defaultPackageName
+	filePath := strings.ToLower(strings.Join(parts[:len(parts)-1], "/"))
```

Comment:
I think we don't need to call strings.ToLower, maybe the user prefers the folder starting with uppercase ~